usaspending_api/etl/management/commands/update_total_funding_amount_sql.py

```diff
@@ -42,8 +42,13 @@ def handle(self, *args, **options):
 
     SUM_TRANSACTION_TFA = """
         With sum_table as (
-        SELECT award_id, SUM(tn.funding_amount) sum_val
-        FROM transaction_normalized as tn join awards a on tn.award_id = a.id WHERE a.id = tn.award_id GROUP BY award_id
+            SELECT award_id, SUM(tn.funding_amount) sum_val
+            FROM transaction_normalized as tn
+            join awards a
+            on tn.award_id = a.id
+            where a.is_fpds = false
+            and tn.funding_amount is not null 
```

Comment:
what're the situations where `funding_amount` would be null here? 

also, `SUM` already ignores nulls. Reference: http://www.postgresqltutorial.com/postgresql-sum-function/

Reply:
There are about 10% of transaction normalized rows that have nulls.  I can dig into why that is, but it is most likely a schema question.

Also, Sum skips nulls, but the join doesnt.  By adding it, we may add speedup/reduce memory.  That being said, I'm playing devils advocate, and it's probably not as valuable as I thought it originally was.